src/index.html

```diff
@@ -1,13 +1,374 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Miami</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Miami</h1>
+  <body class="page__body">
+    <header class="page__section header">
+      <div class="container">
+        <div class="header__content">
+          <div class="top-actions">
+            <a href="#menu" class="icon icon--menu"></a>
+            <a href="/" class="logo">
```

Comment:
Checklist 5th and 6th items, that's what I see wen I click on logo, fix it everywhere
![image](https://user-images.githubusercontent.com/37161315/170691851-ec940d92-f64c-4598-8630-d2a5d5adba25.png)
```suggestion
            <a href="#" class="logo">
```